.cirun.yml

```diff
@@ -0,0 +1,8 @@
+runners:
+  - name: long-runner
+    cloud: aws
+    instance_type: t2.large
+    machine_image: ami-06fd8a495a537da8b
```

Comment:
Also, this AMI doesn't seems to have docker (I tried in my fork), try this: `ami-0048c368c80e3ee99` rather

Reply:
```suggestion
    machine_image: ami-0048c368c80e3ee99
```